### DISCUSSION TITLE
Are you planning to include Mercadopago gateway? (Latin America)

### DISCUSSION THREAD

**Discussion by kurkop** [swellstores/community#538] [Help] [answered]
I really want to implement swell for my store, but there are no gateways available for latin america. I propose Mercadopago because it is the largest. Could I help you to deploy this gateway?

**Comment by ericingram** [comment, marked as answer]
We'll soon support payment app extensions, and then we'll see a lot more international payment gateways added.

Reach out to our team if you're interesting in building it!

**Reply by kurkop** [reply]
Hello @ericingram, what is the status of this? how can I help building this gateway?. Regards

**Reply by ericingram** [reply]
Hey @kurkop we got slowed down on the next major release, but we expect to pick up the pace on this after Storefront V2 is released in April this year.
